kahuna/public/js/usage-rights/usage-rights-editor.html

```diff
@@ -39,94 +38,113 @@
+                                        <label
+                                            for="{{o}}-{{property.name}}-radio-list__item"
+                                            class="radio-list__label"
+                                            ng:class="{'radio-list--selected': ctrl.model[property.name] === o}">
+                                            <div class="radio-list__selection-state"></div>
+                                            <div class="radio-list__label-value">{{ o }}</div>
+                                        </label>
+
+                                    </div>
                                 </div>
-                            </div>
-
-                            <select
-                                class="full-width"
-                                name="{{ property.name }}"
-                                ng:switch-when="true"
-                                ng:model="ctrl.model[property.name]"
-                                ng:required="property.required"
-                                ng:options="o for o in property.options">
-                                <option value="">None</option>
-                            </select>
+
+                                <select
+                                    class="full-width"
+                                    name="{{ property.name }}"
+                                    ng:switch-when="true"
+                                    ng:model="ctrl.model[property.name]"
+                                    ng:required="property.required"
+                                    ng:options="o for o in property.options">
+                                    <option value="">None</option>
+                                </select>
+                        </div>
+
+                        <select
+                            class="full-width"
+                            name="{{ property.name }}"
+                            ng:if="property.optionsMap"
+                            ng:model="ctrl.model[property.name]"
+                            ng:required="property.required"
+                            ng:options="o for o in ctrl.getOptionsFor(property)">
+                            <option value="">None</option>
+                        </select>
                     </div>
 
-                    <select
-                        class="full-width"
+                    <textarea
+                        class="form-input-text"
                         name="{{ property.name }}"
-                        ng:if="property.optionsMap"
                         ng:model="ctrl.model[property.name]"
-                        ng:required="property.required"
-                        ng:options="o for o in ctrl.getOptionsFor(property)">
-                        <option value="">None</option>
-                    </select>
-
+                        ng:switch-when="text"
+                        ng:required="property.required">
```

Comment:
Stupid question: having this whitespace in the textarea doesn't affect its initial value, or does it?